manager/accounts/models.py

```diff
@@ -291,6 +291,12 @@ class AccountTier(models.Model):
         help_text="The maximum storage in project snapshots (GB).",
     )
 
+    file_downloads_month = models.FloatField(
+        verbose_name="File downloads",
+        default=10,
+        help_text="The maximum total size of downloads of project files per month (GB).",
```

Comment:
Possibly a naive question, but what do you think about having the configurations being in bytes and then rendering them using [`filesizeformat` filter](https://docs.djangoproject.com/en/3.0/ref/templates/builtins/#filesizeformat).
Might be a better way for doing this, but my goal would be to allow us finer granularity for configuring quotas per account tier. For example I faced issues already when trying to set `0.5GB` for project storage in production.

Reply:
Thanks. Yes, I recall we maybe had in like that before, and I can't remember why I thought this way was better (maybe I was worried about overflow??). `FloatField` should allow for almost infinite granularity so if it doesn't then we should fix that bug, rather than changing the units.

Reply:
Having thought a little more about this - I think I went for using units of GB in the account tiers just to avoid confusion for us when setting those limits e.g. between bytes, GB, GiB - i.e. just set them at what the user will see, rather than some other value. Confirm that in prod can not set fractional GB for these float fields - may be related to migrations e.g could have been an `IntegerField` before (will test with `file_downloads_month` once deployed).